Makefile

```diff
@@ -76,7 +76,7 @@ builder:
 	IMAGE_TAG=latest make -C infrastructure/docker/builder build
 
 kuberpult:
-	make -C pkg/api all
+	make -C services/frontend-service src/api/api.ts
```

Comment:
Should we change the name of the `src/api/api.ts` target to a something like `build-protos`/`build-deps`?